test/e2e/specs/editor/various/multi-block-selection.spec.js

```diff
@@ -372,7 +372,7 @@ test.describe( 'Multi-block selection (@firefox, @webkit)', () => {
 		const { height } = await paragraphBlock.boundingBox();
 		await paragraphBlock.click( { position: { x: 0, y: height / 2 } } );
 		await paragraphBlock.click( {
-			position: { x: 20, y: height / 2 },
+			position: { x: 25, y: height / 2 },
```

Comment:
Changing this solved the e2e test failure. I'm not exactly sure why the mouse need to move a little further to select the right characters (only in webkit) in this PR. but this unblocks this PR.